CrowsZA/functions/fn_fireSupport.sqf

```diff
@@ -0,0 +1,205 @@
+/*/////////////////////////////////////////////////
+Author: Radkewolf
+			   
+File: fn_fireSupport.sqf
+Parameters: pos
+Return: none
+
+Sets off firesupport in the targeted area
+
+*///////////////////////////////////////////////
+params [["_pos",[0,0,0],[[]],3], ["_logic",objNull,[objNull]]];
+
+// Validate module target
+scopeName "Main";
+private _fnc_errorAndClose = {
+    params ["_msg"];
+    diag_log _msg;
+    breakOut "Main";
+};
+
+// open dialog
+//ZEN
+private _onConfirm =
+{
+	params ["_dialogResult","_in"];
+	_dialogResult params
+	[
+		"_type",
+		"_customType",
+		"_radius",
+		"_seconds",
+		"_delay",
+		"_salvos",
+		"_guns"
+	];
+
+	//Get in params again
+	_in params [["_pos",[0,0,0],[[]],3], ["_logic",objNull,[objNull]]];
+
+	//Check if placed on an object
+	_create = isNull _logic;
+
+	//Check if object is firesupport module
+	if(!_create)
+	then
+	{
+		if(!(_logic isKindOf "Logic"))
+		then
+		{
+			_create = true;
+		}
+		else {
+			_mod = _logic getVariable ["_module", ""];
```

Comment:
When setting variables on units please make sure to add the mod prefix to their name. As other mods or scripts could potentially use that name too. It is just best practice, so calling it: ``getVariable ["crowsZA_module", ""];`` is more than enough.